README.md

```diff
@@ -289,8 +290,8 @@ Running this example produces the following output:
 [Handling Events]: https://ratatui.rs/concepts/event-handling/
 [Layout]: https://ratatui.rs/how-to/layout/
 [Styling Text]: https://ratatui.rs/how-to/render/style-text/
-[templates]: https://github.com/ratatui-org/templates
-[Examples]: https://github.com/ratatui-org/ratatui/tree/main/examples
+[templates]: https://github.com/ratatui-org/templates/
+[Examples]: https://github.com/ratatui-org/ratatui/tree/main/examples/README.md
```

Comment:
An idea for discussion? We can make this line point to the latest release:

```suggestion
[Latest Release]: https://github.com/ratatui-org/ratatui/releases/stable
```

Then we add a link in the release notes:

<img width="666" alt="image" src="https://github.com/ratatui-org/ratatui/assets/1813121/cb78cba8-48b4-4f0b-a8a9-bfec3864c522">

Reply:
> Alternatively, we can link `https://github.com/ratatui-org/ratatui/tree/v0.25.0/examples` here

That was though process too in #779, it's not that hard to maintain, a quick `grep` over the previous version would show where to make changes pretty easily.

Reply:
Why not put that link at the top of the examples README - before even the warning...?
I don't want the links to the examples pointing at release versions as those links go stale. I'd much prefer people see up to date examples, with the option to go to previous versions, than out of date with the option to go to updated versions.

Reply:
(Oh and yes to putting links to the examples in the release notes definitely - it's worth updating those retrospectively even)

Reply:
https://github.com/ratatui-org/ratatui/pull/844/commits/1339d526001973462863c539d63b390834e1aa74

<img width="1341" alt="image" src="https://github.com/ratatui-org/ratatui/assets/381361/0d78641c-0492-4daa-bbd8-1d0a6edd0de5">